61_UI/include/nbl/this_example/common.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef __NBL_THIS_EXAMPLE_COMMON_H_INCLUDED__
+#define __NBL_THIS_EXAMPLE_COMMON_H_INCLUDED__
+
+#include <nabla.h>
+
+// common api
+#include "CCamera.hpp"
+#include "SimpleWindowedApplication.hpp"
```

Comment:
why do you use such weird path as `include/nbl/this_example` ?

Reply:
could just be under the example, or just `include`

Reply:
bump

Reply:
resolved